### PR TITLE
IP executer is being treated as a CC executer

### PR DESCRIPTION
**Pull request by 6hundreds**
Part of https://github.com/gradle/gradle/issues/29045

### CODE REVIEW

**Comment by 6hundreds** [comment]
@bot-gradle test this without PTS

**Comment by 6hundreds** [comment]
@bot-gradle test acc

**Comment by 6hundreds** [comment]
@bot-gradle test acc

**Comment by 6hundreds** [comment]
@bot-gradle test this

File: testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/GradleContextualExecuter.java [hunk 1]
```diff
@@ -84,7 +84,8 @@ public static boolean isNotConfigCache() {
     }
 
     public static boolean isConfigCache() {
-        return getSystemPropertyExecuter() == Executer.configCache;
+        return getSystemPropertyExecuter() == Executer.configCache ||
```
Comment (bamboo) [inline testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/GradleContextualExecuter.java:87]: 💅 Introduce local variable for `getSystemPropertyExecuter()` lookup

**Review by bamboo — APPROVED** [review]
👍

**Pull request merged**